src/UI/UIUtils.java

```diff
@@ -260,8 +270,53 @@ public static COSpriteSpec instance(Commander co)
         spec = new COSpriteSpec(co.faction, co.myColor);
       return spec;
     }
+
+    /**
+     * Attempts to pull the key color for this canon faction from loaded resources.
+     * <p>Falls back on Salmon
+     */
+    public static COSpriteSpec fromDisk(String col, String fac)
+    {
+      initCosmetics();
+      COSpriteSpec spec = null;
+      final Faction facInst = new Faction(fac);
+
+      for( Entry<Color, String> pair : paletteNames.entrySet() )
+      {
+        if( pair.getValue().equalsIgnoreCase(col) )
+        {
+          spec = new COSpriteSpec(facInst, pair.getKey());
+        }
+      }
+
+      if( null == spec )
+        spec = new COSpriteSpec(facInst, Color.PINK);
```

Comment:
We should probably create a custom color constant (a la `PURPLE`) for this to call it `ROSE` instead to avoid confusion with the external pink.
For that matter, `PURPLE` should probably be renamed to `VIOLET` since that's what we actually call it.